engine/settings/base.py

```diff
@@ -443,6 +480,10 @@ class BrokerTypes:
     "ALERT_GROUP_ESCALATION_AUDITOR_CELERY_TASK_HEARTBEAT_URL", None
 )
 
+BEAT_SCHEDULE_DB_FILENAME = os.getenv("BEAT_SCHEDULE_DB_FILENAME", "celerybeat-schedule")
+if BEAT_SCHEDULE_DB_FILENAME:
+    CELERY_BEAT_SCHEDULE_FILENAME = BEAT_SCHEDULE_DB_FILENAME
```

Comment:
is there a need for the `if` clause here?

```suggestion
CELERY_BEAT_SCHEDULE_FILENAME = os.getenv("CELERY_BEAT_SCHEDULE_FILENAME", "celerybeat-schedule")
```

Reply:
I will fix it.

TLDR (and link for history): https://docs.celeryq.dev/en/stable/userguide/configuration.html#new-lowercase-settings